static/js/recent_topics.js

```diff
@@ -316,6 +316,17 @@ function format_topic(topic_data) {
     const all_senders = recent_senders.get_topic_recent_senders(stream_id, topic);
     const senders = all_senders.slice(-MAX_AVATAR);
     const senders_info = people.sender_info_for_recent_topics_row(senders);
+    const other_sender_ids = all_senders.slice(0, -MAX_AVATAR);
+    let other_senders_fullname_string = "";
+    for (const id of other_sender_ids) {
+        if (muting.is_user_muted(id)) {
+            other_senders_fullname_string = other_senders_fullname_string.concat("Muted user, ");
+        } else {
+            const fullname = people.get_full_name(id);
```

Comment:
I think we can use `people.get_display_full_names` here and avoid some duplication.

Reply:
Yeah, we should definitely do that.

Reply:
Made these changes.